src/sql.rs

```diff
@@ -717,7 +717,13 @@ pub async fn housekeeping(context: &Context) -> Result<()> {
     // only works if auto_vacuum is enabled.
     match context
         .sql
-        .query_row_optional("PRAGMA incremental_vacuum", (), |_row| Ok(()))
+        .call_write(move |conn| {
+            match conn.query_row("PRAGMA incremental_vacuum", (), |_row| Ok(())) {
```

Comment:
I would expect that it's easier to "just" use `execute` instead of `call_write` and `query_row`?